UltraStar Play/Assets/Common/Model/ESongSelectOrder.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
all those using includes seem unnecessary / unused for the enum.

Reply:
true